### DISCUSSION TITLE
Adopt the approach of Wispr Flow

### DISCUSSION THREAD

**Discussion by worldoptimizer** [EpicenterHQ/epicenter#329] [Ideas]
I just started using https://www.flowvoice.ai/, and I must say I would love for Whispering to move in that direction. The way it integrates into the system is much easier to use, and perhaps that could be adopted here as well. I'd gladly move back and pay a monthly fee.

**Comment by braden-w** [comment]
Great suggestion! And funny coincidence haha, I also recently stumbled into it and I've tried it for the past few days! Will aim for a few features. Whispering is in a similar space and could learn a few improvements.

The biggest quality of life update I'm working on is #307 , allowing us the additional option to hold to talk, and #198 , post processing.
